### PR TITLE
Updates blueprints to follow ember-suave

### PR DESCRIPTION
**Pull request by xcambar**
Small updates to the app blueprint, for a happier ember-suave.

### CODE REVIEW

**Comment by stefanpenner** [comment]
babel is now mandatory on master. So i believe this can be merged.

r? @chadhietala 
r? @rwjblue

**Comment by chadhietala** [comment]
Seems good. Test failure is racy `npm pack` test.

**Comment by stefanpenner** [comment]
> Seems good. Test failure is the race npm pack test.

Ya I'm sick of that one, I'll fix it soon (once i get some sleep)

File: blueprints/app/files/tests/helpers/start-app.js [hunk 1]
```diff
@@ -3,9 +3,9 @@ import Application from '../../app';
...
 
 export default function startApp(attrs) {
-  var application;
+  let application;
 
-  var attributes = Ember.merge({}, config.APP);
+  let attributes = Ember.merge({}, config.APP);
   attributes = Ember.merge(attributes, attrs); // use defaults, but you can override;
 
   Ember.run(function() {
```
Comment (stefanpenner) [inline blueprints/app/files/tests/helpers/start-app.js]: `() => {`

Comment (xcambar) [inline blueprints/app/files/tests/helpers/start-app.js]: done. Shall I squash?

**Comment by xcambar** [comment]
@stefanpenner Done.

File: blueprints/app/files/app/app.js [hunk 2]
```diff
@@ -3,14 +3,12 @@ import Resolver from 'ember-resolver';
 import loadInitializers from 'ember/load-initializers';
 import config from './config/environment';
 
-var App;
-
 Ember.MODEL_FACTORY_INJECTIONS = true;
 
-App = Ember.Application.extend({
+const App = Ember.Application.extend({
```
Comment (kellyselden) [inline blueprints/app/files/app/app.js]: definition was moved above by a PR, someone's linter complained #3793. I'm for putting it back to one line, just letting you know reasoning.

File: blueprints/app/files/tests/helpers/start-app.js [hunk 3]
```diff
@@ -3,12 +3,12 @@ import Application from '../../app';
...
 export default function startApp(attrs) {
-  var application;
+  let application;
 
-  var attributes = Ember.merge({}, config.APP);
+  let attributes = Ember.merge({}, config.APP);
   attributes = Ember.merge(attributes, attrs); // use defaults, but you can override;
 
-  Ember.run(function() {
+  Ember.run(()=> {
```
Comment (kellyselden) [inline blueprints/app/files/tests/helpers/start-app.js]: whitespace () =>

**Comment by kellyselden** [comment]
I added some comments.

**Comment by xcambar** [comment]
@kellyselden ember-suave doesn't complain, therefore I don't :) I applied your comments.

**Comment by stefanpenner** [comment]
rebase, master should now be green.

File: blueprints/app/files/app/app.js [hunk 4]
```diff
@@ -3,14 +3,14 @@ import Resolver from 'ember-resolver';
 import loadInitializers from 'ember/load-initializers';
 import config from './config/environment';
 
-var App;
+let App;
 
 Ember.MODEL_FACTORY_INJECTIONS = true;
 
 App = Ember.Application.extend({
```
Comment (xtian) [inline blueprints/app/files/app/app.js]: I think this would be better as `const App = Ember.Application.extend({`

**Comment by xcambar** [comment]
:sunglasses: thanks for #4887 @stefanpenner!

**Comment by kellyselden** [comment]
@homu r+

**Comment by stefanpenner** [comment]
@homu retry

**Pull request merged**